resources/i18n/pl_PL/fdmprinter.def.json.po

```diff
@@ -3817,7 +3817,7 @@ msgstr "Przyczepność"
 #: fdmprinter.def.json
 msgctxt "prime_blob_enable label"
 msgid "Enable Prime Blob"
-msgstr "Włącz Czyszcz. \"Blob\""
+msgstr "Włącz Czyszczenie \"Blob”ów"
```

Comment:
If this was difficult to translate, maybe we should translate the word "bubble" or "droplet".